rj_gameplay/classifier/fumble_classifier.py

```diff
@@ -0,0 +1,123 @@
+import stp.rc as rc
+import rj_gameplay.gameplay_node as gameplay_node
+import rclpy
+
+
+def fumble_classifier(world_state: rc.WorldState, possess) -> bool:
+	if possess is None:
+		return True
+	else: 
+		return False
+
+"""
+get_distance returns the distance between the ball and the kicker
+num: kicker's id
```

Comment:
Nit: `num` -> `kicker` (or rename the parameter and the doc to `kicker_id`, which is probably the ideal name)